Microsoft Windows [Versión 10.0.19044.1466]
(c) Microsoft Corporation. Todos los derechos reservados.

C:\Users\MX-30>python -m venv env

C:\Users\MX-30>source env/bin/activate
"source" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

C:\Users\MX-30>env/bin/activate
"env" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

C:\Users\MX-30>env\bin\activate
El sistema no puede encontrar la ruta especificada.

C:\Users\MX-30>env\Scripts\activate

(env) C:\Users\MX-30>pip freeze

(env) C:\Users\MX-30>pip install python-dateutil
Collecting python-dateutil
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting six>=1.5
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: six, python-dateutil
Successfully installed python-dateutil-2.8.2 six-1.16.0
WARNING: You are using pip version 21.2.4; however, version 22.0.3 is available.
You should consider upgrading via the 'C:\Users\MX-30\env\Scripts\python.exe -m pip install --upgrade pip' command.

(env) C:\Users\MX-30>pip freeze
python-dateutil==2.8.2
six==1.16.0

(env) C:\Users\MX-30>desactivate
"desactivate" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

(env) C:\Users\MX-30>deactivate
C:\Users\MX-30>
C:\Users\MX-30>
C:\Users\MX-30>